src/Arriba/Arriba.Server/Controllers/OAuthController.cs

```diff
@@ -94,7 +96,7 @@ private async Task<OAuthTokenResult> ReadTokenResultAsync(Uri uri, HttpContent c
             var response = await _http.PostAsync(uri, content);
             var result = await response.Content.ReadAsStringAsync();
 
-            _log.WriteLine($"ReadTokenResultAsync result: {result}");
+            _log.WriteLine("ReadTokenResultAsync result: {0}", result);
```

Comment:
I did that before.. by adding the WriteLine(string x), because I was getting a Error during the runtime (and just at runtime) it tries to use the params with 0 items on it. 

I thought that you wanted to change it to use what was available on _log.. I will redo the changes  ;)